packages/manager/package.json

```
@@ -2,7 +2,7 @@
   "name": "linode-manager",
   "author": "Linode",
   "description": "The Linode Manager website",
-  "version": "1.148.0",
+  "version": "1.148.1",
```

We can keep the version as 

```suggestion
  "version": "1.148.0",
```